app/Services/GocardlessMapper.php

```
@@ -60,47 +59,207 @@ private function get(array $array, string $key, $default = null)
+            'metadata' => $metadata,
+            'import_data' => json_encode($transaction, JSON_THROW_ON_ERROR | JSON_UNESCAPED_UNICODE),
         ];
+    }
+
+    /**
+     * Determines the transaction type based on the transaction data.
+     *
+     * @param array $transaction The transaction data.
+     * @return string The determined transaction type.
+     */
+    private function determineTransactionType(array $transaction): string
```

PHPStan (2.1.15)

Method App\Services\GocardlessMapper::determineTransactionType() has parameter $transaction with no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)